app/config/index.js

```diff
@@ -5,7 +5,7 @@ import colors from './colors'
 
 let config = {
   colors,
-  storybookEnabled: false,
+  storybookEnabled: true,
```

Comment:
This should be in the `local.js` config as documented in the README. We don't want Storybook to be true and committed to the repo - it will then impact other contributors.